### ISSUE TITLE
Allow 

### ISSUE THREAD

**Issue by WilsonNet** [3ba2ii/LeetSync#118]
**Is your feature request related to a problem? Please describe.**
Right now the extensions ask for complete permission of my github account including the repos that I have access in the organizations I'm part of, that is too aggressive.

**Describe the solution you'd like**
Allow the user to control the scope of the permission, at least give permission just to selected repos.

**Describe alternatives you've considered**
Avoid permission to leak to organizations and keep only in the user scope (I don't know if this is possible).

**Additional context**
Too much permission to use this extension, if someone with malicious intent take control of the extension it can push an update that may cause serious destruction.

**Comment by ajmedeio** [comment]
I agree entirely. I can't use this app until permissions are constrained to a single repository.
